schema.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(225),
+    date_of_birth DATE
+);
+CREATE INDEX idx_patients_id ON patients(id); 
+
+
+CREATE TABLE medical_histories(
+    id INT  PRIMARY KEY,
+    admited_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(225)
+);
+CREATE INDEX idx_medical_histories_id ON medical_histories(id); 
+
+
+CREATE TABLE invoices (
+    id INT PRIMARY KEY,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT REFERENCES medical_histories(id)
+);
+CREATE INDEX idx_invoices_id ON invoices(id); 
+
+CREATE TABLE treatments(
+    id INT PRIMARY KEY,
+    type VARCHAR(225),
+    name VARCHAR(225)
+);
+CREATE INDEX idx_treatments_id ON treatments(id);
+
+
+CREATE TABLE  invoice_items (
+   id INT PRIMARY KEY,
+   unit_price DECIMAL,
+   quantity INT,
+   total_price DECIMAL,
+   invoice_id INT REFERENCES envoices(id),
+   traitment_id  INT REFERENCES traitments(id)
```

Comment:
- [OPTIONAL]  Well done for implementing the optional suggestion highlighted by the first reviewer 👏. However, when I am trying to test your current implementation, I'm encountering the following errors ⚠. These issues arise because you may have forgotten to update the name here. Since this was missed by the previous reviewer, I am making it optional; however, I highly recommend fixing them. Besides this, your implementation, including join table, is working as expected, kudos 🎉🚀

![image](https://github.com/DjibrilM/clinic_database/assets/106119105/cfb2d5ac-2c49-4cd1-999a-7d6c9e84968d)
![image](https://github.com/DjibrilM/clinic_database/assets/106119105/8b1d356f-f093-4f91-8035-4bfed6652064)